images/router/haproxy/conf/haproxy-config.template

```diff
@@ -328,6 +328,23 @@ backend be_tcp_{{$cfgIdx}}
   timeout tunnel  {{$value}}
       {{ end }}
     {{ end }}
+
+{{ if matchPattern "true|TRUE" (index $cfg.Annotations "haproxy.router.openshift.io/rate-limit-connections") }}
```

Comment:
Nit (should not block merge): indentation appears off here (should match the same level as immediately above)